src/test/java/seedu/task/testutil/TypicalTestTasks.java

```diff
@@ -55,7 +60,12 @@ public static void loadTaskManagerWithSampleData(TaskManager ab) {
 
     public TaskManager getTypicalTaskManager(){
         TaskManager ab = new TaskManager();
-        loadTaskManagerWithSampleData(ab);
+        try {
+            loadTaskManagerWithSampleData(ab);
+        } catch (IllegalValueException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
```

Comment:
Can change to `assert false: "reason"` instead? Because this exception will not happen unless someone happens to make the wrong sample tasks.